app/controllers/charges_controller.rb

```diff
@@ -91,4 +91,55 @@ def generate_description(seller_names:, item_types:)
 
     description
   end
+
+  def create_square_payment_request(source_id:, amount:, note:, email:, line_items:)
+    api_client = Square::Client.new(
+        access_token: ENV['SQUARE_ACCESS_TOKEN'],
+    )
+
+    request_body = {
+        source_id: source_id,
+        idempotency_key: SecureRandom.uuid,
+        amount_money: {
+            amount: amount,
+            currency: 'USD',
+        },
+        buyer_email_address: email,
+        note: note,
```

Comment:
Do you know if note gets emailed to the buyer?

Reply:
docs say "An optional note to be entered by the developer when creating a payment

Limit 500 characters."

so seems like no - so we can add that in later if we'd like! I'll add a tentative TODO

Reply:
Sounds good! Yeah we need it to appear on the receipt. I have a feeling we might have to create an "Order"